src/main/kotlin/no/nav/familie/tilbake/api/PerioderController.kt

```diff
@@ -38,10 +50,67 @@ class PerioderController(
     )
     fun erPerioderLike(
         @PathVariable behandlingId: UUID,
-    ): Ressurs<Boolean> =
-        Ressurs.success(
+    ): Ressurs<Boolean> {
+        val fagsak = fagsakRepository.finnFagsakForBehandlingId(behandlingId)
+        val erEnsligForsørgerOgPerioderLike =
             faktaFeilutbetalingService.sjekkOmFaktaPerioderErLike(behandlingId) &&
                 foreldelseService.sjekkOmForeldelsePerioderErLike(behandlingId) &&
-                vilkårsVurderingService.sjekkOmVilkårsvurderingPerioderErLike(behandlingId),
+                vilkårsVurderingService.sjekkOmVilkårsvurderingPerioderErLike(behandlingId) &&
+                fagsak.ytelsestype.tilTema() == Tema.ENF
+
+        val vedtaksbrevsoppsummering = vedtaksbrevsoppsummeringRepository.findByBehandlingId(behandlingId)
+        if (!erEnsligForsørgerOgPerioderLike && vedtaksbrevsoppsummering != null) {
+            vedtaksbrevsoppsummeringRepository.update(vedtaksbrevsoppsummering.copy(skalSammenslåPerioder = false))
```

Comment:
Er litt skeptisk til at en "GET"-er oppdaterer en verdi. 
Bør denne sjekken og oppdateringen utføres når vilkårsvurderingen lagres ned - gitt at det eksisterer en vedtaksbrevoppsummering for behandlingen?